src/sentry/utils/snuba.py

```diff
@@ -639,6 +640,10 @@ def raw_query(
         is_grouprelease=is_grouprelease,
         **kwargs,
     )
+
+    if should_use_snql(referrer):
+        use_snql = True
```

Comment:
I don't quite get why we pass an argument `use_snql` into this function then discard it's value only if `should_use_snql` returns true but keep its value if false. The referrer list hardcoded in the snql module is also separate to the referrers that might actually pass `use_snql=True`, and it's unclear to me why there's two ways to make a referrer potentially use snql.

Reply:
Right now it's there for testing purposes and maybe forward compatibility.

Reply:
That makes sense. I'll change it to be more intuitive.